### ISSUE TITLE
viewing n-dimensional array

### ISSUE THREAD

**Issue by d-v-b** [flika-org/flika#45]
Light sheet microscopy often generates 4D datasets, where a single timepoint is a volume. It would be awesome if flika could support these datasets (and potentially 5-dimensional data, e.g. [time, trial, z, y, x]). 
Supporting these data would require a) generating an axis slider for each non-color axis beyond the first 2, and b) some way for the user to define which axis is time. 

I'm not familiar enough with flika/pyqtgraph internals to submit a PR for this yet, but if there was a working PR for this would it be something the flika devs would be interested in?

**Comment by kyleellefsen** [comment]
Yes, we would definitely be interested in a 4D viewer. I did write a "Volume_Viewer" class in the flika plugin [light_sheet_analyzer](https://github.com/kyleellefsen/light_sheet_analyzer/blob/master/light_sheet_analyzer.py) that has some of this functionality: you can pick which of the two 'x, y, z' axes you want to view as the 'x, y' axes on the image. I wasn't happy enough with this implementation to put it into flika; the interface felt clunky and I've seen much better volume viewers. 

But yes, if someone submitted a PR with a better general purpose 4D volume viewer (or 5D viewer), I think it would be useful to a lot of users.
